### DISCUSSION TITLE
关于umi生成的tsconfig.json文件配置，能在umi配置之中对其进行修改吗？

### DISCUSSION THREAD

**Discussion by ByYogi** [umijs/umi#10252] [Q&A] [answered]
我根目录有个 tsconfig.json 文件，
``` json
{
 "extends": "./src/.umi/tsconfig.json",
}
```
我想拓展一些其他配置
``` json
{
  "extends": "./src/.umi/tsconfig.json",
  "compilerOptions": {
	  "paths": {
		  "@test/*": [
			"src/pages/test/*"
		  ],
	  }
  }
}
```
发现该配置会覆盖`extends`中的paths，而不是合并。
所以，我想知道`.umirc.ts`文件是否有能影响自生成的tsconfig相关api配置

**Comment by fz6m** [comment, marked as answer]
按设计目前不可以自己改 alias 。有一个解决方案，在 #8376 讨论吧。

**Reply by ByYogi** [reply]
目前在配置中使用`alias`可实现别名路径。
``` ts
{
  alias: {
      '@test': '/src/pages/test'
  }
}
```
---
因为没有注入到生成的`tsconfig.json`文件，编译器无法识别，但是能正常跑 #8376 。等待解决方案
![image](https://user-images.githubusercontent.com/111735598/211453326-b6071e36-c8ae-4044-93b8-565f519f23a9.png)
